### PR TITLE
Remove the tus chunk endpoints from the schema

### PR DESCRIPTION
**Pull request by SpecLad**
### Motivation and context

As explained in the comment in `_tus_chunk_action`, I don't think we need explicit documentation for these endpoints. Hiding them makes the documentation a bit cleaner, especially because these endpoints were indistinguishable from one another.

It also removes the corresponding SDK code, which is good, since clients should follow the URL returned by the creation endpoint, not invoke the chunk endpoints directly.

### How has this been tested?

CI.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- ~~[ ] I have updated the documentation accordingly~~
- ~~[ ] I have added tests to cover my changes~~
- ~~[ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- ~~[ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat/apps/engine/views.py [hunk 1]
```diff
@@ -83,6 +83,26 @@
 
 _UPLOAD_PARSER_CLASSES = api_settings.DEFAULT_PARSER_CLASSES + [MultiPartParser]
 
+def _tus_chunk_action(*, detail: bool, suffix_base: str):
```
Comment (zhiltsov-max) [inline cvat/apps/engine/views.py:86]: I suggest putting this function to `view_utils.py`

Comment (SpecLad) [reply cvat/apps/engine/views.py]: Alright, done.

**Review by zhiltsov-max — APPROVED** [review]

File: cvat/apps/engine/views.py (hunk 1 above)
Comment (SpecLad) [reply cvat/apps/engine/views.py]: This caused an import loop, so I moved some stuff around to fix it.

**Pull request merged**